modin/scripts/scripts.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+import click
+
+from modin.scripts import cluster
+
+
+@click.group()
+def cli():
+    pass
+
+
+@click.command()
+@click.option(
+    "--config",
+    required=True,
+    type=str,
+    help="the config file for the cluster")
+@click.option(
+    "--port",
+    required=True,
+    help="port to which to forward the notebook server")
+def notebook(config, port):
+    config = cluster.load_config(config)
+    cluster.validate_config(config)
+    print("\nSetting up cluster\n")
+    redis_address = cluster.setup_cluster(config)
+    print("\nLaunching notebook\n")
+    print("*" * 68)
+    print(("To connect to the cluster, run the following commands in the "
```

Comment:
Ideally this would go in our `__init__` so we should be able to detect if the cli was used.

Reply:
Agreed, I'm still thinking about how to best set this up. Do you think setting an environment variable with the redis address and detecting that in the `__init__` is a good solution?

Reply:
That would probably be good, but we probably should use two environment variables, one for `MODIN_EXECUTION_FRAMEWORK` and one for `MODIN_RAY_REDIS_ADDRESS` or something like that.